vmm/src/vm_config.rs

```diff
@@ -645,4 +796,94 @@ pub struct VmConfig {
     // valid, and will be closed when the holding VmConfig instance is destroyed.
     #[serde(skip)]
     pub preserved_fds: Option<Vec<i32>>,
+    #[serde(default)]
+    pub landlock_enable: bool,
+    pub landlock_config: Option<Vec<LandlockConfig>>,
```

Comment:
I just realized these new options need to be added to our openapi specification file: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/vmm/src/api/openapi/cloud-hypervisor.yaml.

Reply:
I will update this

Reply:
What is the value of adding landlock options to openapi spec? Does the API framework validate inputs to API request?

I ask this because, without any API spec changes I was able to pass `landlock_enable` and `landock_config` options to `vm.create` request.

Reply:
The openapi spec file is the way how any other services get to know the RESTful APIs (and their formats) provided by Cloud Hypervisor. Without the knowledge of Cloud Hypervisor internals (e.g. API options and data structures), other services has no way to use Cloud Hypervisor RESTful APIs correctly. Kata containers now is the primary user of this spec file.

Reply:
Understood, thanks for the details.